profiles/api/oauth2.py

```diff
@@ -136,4 +138,14 @@ def token() -> Response:
     filtered_json_data = {your_key: json_data[your_key] for your_key in
                           ['access_token', 'expires_in', 'name', 'orcid', 'token_type']}
```

Comment:
is `name` a placeholder or undocumented? https://members.orcid.org/api/get-oauthtoken doesn't list it, if it's the correct place to look?

Reply:
Interesting, it's being returned but is undocumented. I think we'll have to use the real API at this point anyway, so can probably ignore it.